pmd-java/src/main/resources/category/java/errorprone.xml

```diff
@@ -2852,16 +2852,67 @@ public class Foo {
         </example>
     </rule>
 
+	<rule name="ReturnEmptyCollectionRatherThanNull"
+          language="java"
+          since="6.37.0"
+          class="net.sourceforge.pmd.lang.rule.XPathRule"
+          message="Return an empty collection rather than 'null'."
+          externalInfoUrl="${pmd.website.baseurl}/pmd_rules_java_errorprone.html#returnemptycollectionratherthannull">
+        <description>
+For any method that returns an collection (such as an array, Collection or Map), it is a better to return an empty one rather than a
```

Comment:
```suggestion
For any method that returns an collection (such as an array, Collection or Map), it is better to return an empty one rather than a
```